### DISCUSSION TITLE
`coredns` toleration for control planes

### DISCUSSION THREAD

**Discussion by dhess** [siderolabs/talos#8324] [Q&A]
It appears there's a toleration in the hard-wired Talos config that permits `coredns` pods to run on control plane nodes. Is it possible to remove that toleration so that they only run on worker nodes? And would there be any repercussions? (I suspect not, since occasionally both `coredns` pods will run on worker nodes.)

**Comment by smira** [comment]
You can disable Talos's CoreDNS manifest and use your own that works for you. Talos provides some defaults, but they don't fit every usecase.
